### PR TITLE
imported a second controller

### PR DESCRIPTION
**Pull request by aebill14**
Just wanted to make sure I had a second controller so I can use the old gunner pannel to put some commands on

### CODE REVIEW

File: src/main/java/frc/robot/RobotContainer.java [hunk 1]
```diff
@@ -39,6 +39,9 @@ public class RobotContainer
   // Replace with CommandPS4Controller or CommandJoystick if needed
   private final CommandXboxController m_driverController =
       new CommandXboxController(OIConstants.kDriverControllerPort);
+
+  private final CommandXboxController m_coDriverController =
+      new CommandXboxController(OIConstants.kCoDriverControllerPort);
```
Comment (jc3m) [inline src/main/java/frc/robot/RobotContainer.java:44]: This is a gunner controller?

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -79,10 +82,25 @@ public RobotContainer()
...
     m_driverController.a().whileTrue(new FI_IntakeForward(floorIntake));
     m_driverController.b().debounce(.1).onTrue(new ArmPositions(arm));
+
+    //Gunner Controls
+    /*
+     *  Moves:
+     *   - Fly Wheel
+     *   - Pivot 
+     */
+    m_coDriverController.button(1);
```
Comment (jc3m) [inline src/main/java/frc/robot/RobotContainer.java:103]: What's the purpose of this line here?

File: src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (aebill14) [reply src/main/java/frc/robot/RobotContainer.java:44]: yes!

File: src/main/java/frc/robot/RobotContainer.java (hunk 2 above)
Comment (aebill14) [reply src/main/java/frc/robot/RobotContainer.java:103]: just to see if implementing just a button would work, should I get rid of it, it was a tester.

Comment (jc3m) [reply src/main/java/frc/robot/RobotContainer.java:103]: Yeah, in general if you can remove test-only code, it'll keep your codebase cleaner

File: src/main/java/frc/robot/RobotContainer.java (hunk 1 above)
Comment (jc3m) [reply src/main/java/frc/robot/RobotContainer.java:44]: Nice!

File: src/main/java/frc/robot/RobotContainer.java (hunk 2 above)
Comment (aebill14) [reply src/main/java/frc/robot/RobotContainer.java:103]: sounds good, sweeping up the trash 🧹

**Review by jc3m — APPROVED** [review]

**Pull request merged**